cmd/flux/bootstrap_git.go

```diff
@@ -116,9 +117,22 @@ func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	gitAuth, err := transportForURL(repositoryURL)
-	if err != nil {
-		return err
+
+	if strings.Contains(repositoryURL.Hostname(), "git-codecommit") && strings.Contains(repositoryURL.Hostname(), "amazonaws.com") {
+		if repositoryURL.Scheme == "ssh" {
```

Comment:
nit: instead of using literals, we could use the schemes defined in the git package: `git.{SSH|HTTP|HTTPS}`.